### ISSUE TITLE
508 Compliance: Attachment and Emoji button missing focus

### ISSUE THREAD

**Issue by kelseygittings** [mattermost/mattermost-server#8362]
Per Mattermost guidelines, GitHub issues are for bug reports: http://www.mattermost.org/filing-issues/

Summary
When using Chrome Web Portal JAWS cannot identify the Attachment or Emoji button.

#### Steps to reproduce
Mattermost Team Edition Version: 4.7.1 (4.7.2-rc3)
Chrome Version 64.0.3282.168 (Official Build) (64-bit)
Steps to recreate:
1. Launch JAWS
2. Launch https://oss.mattermost.com/joomla/channels/town-square
3. JAWS will read the page. It will only read, "Form submission: Write a message..."
4. Click in the message box.
5. JAWS will read the same answer as step 3.
6. Hit Tab on your keyboard.
7. JAWS will not tell the user what the button is, what it does, or highlight where it is.

#### Expected behavior
JAWS should be able to read every navigation option including buttons on the page. When the user hits tab JAWS should say something to the effect of, "Attachment button, Press Space Bar to Confirm."

#### Observed behavior (that appears unintentional) 
For 508 compliance, each interactive element should have visible focus when the user tabs to the element using the keyboard, and JAWS should be able to tell the user what each element does.

Currently, the user is not able to determine where they are while tabbing because from a lack of visible indicator, and for the seeing impaired it does not even tell them what the button is.
![mattermost-508-compliance](https://user-images.githubusercontent.com/36772251/36609471-882287f2-1892-11e8-9cc1-d6684d61294f.png)

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem.
This needs code and libraries added, not just a few lines that need fixing.

**Comment by santos22** [comment]
Hi @kelseygittings - I think this might be related https://github.com/mattermost/mattermost-server/issues/8034 👍

**Comment by lindy65** [comment]
Thanks for the feedback @kelseygittings :)

As `santos22` indicates above, there is an open ticket to implement this compliance feature so I'll close off this GitHub issue for now...
